controllers/likedVideos.controller.js

```diff
@@ -0,0 +1,53 @@
+const { User } = require("../models/user.model");
+
+const getLikedVideos = async (req, res) => {
+  try {
+    const result = await User.findOne({}).populate("likedVideos.videoId");
+    res.status(200).json({ success: true, likedVideos: result.likedVideos });
+  } catch (err) {
+    res.status(500).json({
+      success: false,
+      message: "Could not fetch your cart",
```

Comment:
The message is wrong here it should be could not fetch likedVideos

Reply:
Oh yeahh ! Copy pasted the catch block . Thanks for pointing that out